.drone.yml

```
@@ -108,3 +108,24 @@ steps:
       VERSION: "${DRONE_TAG}"
     depends_on:
       - clone
+
+  - name: deploy migrator to cs-prod
+    image: quay.io/ukhomeofficedigital/kd
+    commands:
+      - cd kube
+      - ./deploy.sh
+    environment:
+      ENVIRONMENT: ${DRONE_DEPLOY_TO}
+      DEPLOYMENT_TYPE: migrator
+      SQS_SECRET_NAME: case-migrator-sqs
+      KUBE_TOKEN:
+        from_secret: hocs_case_creator_${DRONE_DEPLOY_TO/-/_}
+      KUBE_SERVER: https://kube-api-prod.prod.acp.homeoffice.gov.uk
+      CLUSTER_NAME: acp-prod
+      VERSION: "${DRONE_TAG}"
+    depends_on:
+      - fetch and checkout
```

I'm not sure this will work: "fetch and checkout" isn't the name of a pipeline nor a step inside the current pipeline (it's in a different pipeline, and I think that's inaccessible?).

```suggestion
      - clone
```

"Clone" is the name of the implicit step that runs before the other pipeline steps, which clones the repository (without tags). Usually you don't have to specify it explicty unless you're doing something clever -- so you might be able to remove the depends_on clause entirely.